api/curve/v1/services/plugin.py

```diff
@@ -206,7 +209,10 @@ def __call__(self, method, *args):
             if not isinstance(plugin.__dict__[method], types.FunctionType):
                 continue
             try:
+                current_app.logger.info('Invoking call with __call__ %s %s', plugin, method)
                 output = self._invoke(plugin, method, *args)
+                current_app.logger.info('Returned %s', output)
+                output = 'default', (2300, 2900)
```

Comment:
What's the advantage for this change? This line will cause an internal error during upload.

Reply:
Apologies. This was for debugging. I'll revise the PR.

Reply:
I have corrected this in the new PR.